app/src/main/java/net/gsantner/markor/activity/DocumentActivity.java

```diff
@@ -202,7 +202,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public boolean dispatchTouchEvent(MotionEvent event) {
-        if (_appSettings.isSwipeToChangeMode() && getCurrentVisibleFragment() instanceof DocumentEditFragment) {
+        if (_appSettings.isSwipeToChangeMode() && _appSettings.isEditorLineBreakingEnabled() && getCurrentVisibleFragment() instanceof DocumentEditFragment) {
```

Comment:
@gsantner I have also added this test to only enable swipe to change mode if line breaking is enabled.

Would you prefer if I flip this logic? i.e. only enable horizontal scrolling if swipe to change mode is disabled?

Reply:
experimental swipe-to-change doesn't work well..whenever hscroll is enabled it's fine that hscroll is enabled and swipe ignored.

Reply:
we can basically also remove swipe and think of other solution...it makes more problems than it tries to resolve at this point

Reply:
In the future, we could probably enable multi-finger gestures. 1 fingers = scroll. 2 fingers = swipe or something like that. The button is always available in the tool bar any way, so swipe is an extra convenience.